src/tokens/ERC20.huff

```diff
@@ -70,7 +70,7 @@
     [INITIAL_DOMAIN_SEPARATOR] sstore       // []
     
     // Copy the runtime bytecode with constructor argument concatenated.
-    0x67                                    // [offset] - constructor code size
+    0x66                                    // [offset] - constructor code size
```

Comment:
```suggestion
    __codesize(CONSTRUCTOR)                 // [offset] - constructor code size
```
still [not perfect](https://github.com/huff-language/huffmate/pull/116/files#r1148161068) but better than hardcoded